src/graph/graph.c

```diff
@@ -669,15 +669,19 @@ void Graph_GetNodeEdges(const Graph *g, const Node *n, GRAPH_EDGE_DIR dir, int e
 
 /* Removes an edge from Graph and updates graph relevent matrices. */
 int Graph_DeleteEdge(Graph *g, Edge *e) {
-	uint64_t x;
-	GrB_Matrix R;
-	GrB_Matrix M;
-	GrB_Info info;
-	EdgeID edge_id;
-	GrB_Matrix TR = GrB_NULL;
-	int r = Edge_GetRelationID(e);
-	NodeID src_id = Edge_GetSrcNodeID(e);
-	NodeID dest_id = Edge_GetDestNodeID(e);
+	uint64_t    x;
+	GrB_Info    info;
+	GrB_Matrix  R        =  GrB_NULL;
+	GrB_Matrix  M        =  GrB_NULL;
+	GrB_Matrix  TR       =  GrB_NULL;
+	EdgeID      edge_id  =  INVALID_ENTITY_ID;
+
+	assert(g != NULL);
+	assert(e != NULL);
```

Comment:
why those assertions?
also why we moved from `assret(g)` to `assret(g != NULL)`?